system/HTTP/Message.php

```diff
@@ -142,7 +142,14 @@ public function getProtocolVersion(): string
 	 */
 	public function isJSON()
 	{
-		return $this->hasHeader('Content-Type')
-			&& $this->header('Content-Type')->getValue() === 'application/json';
+		if (! $this->hasHeader('Content-Type'))
+		{
+			return false;
+		}
+
+		$header = $this->header('Content-Type')->getValue();
+		$parts  = explode(';', $header);
+
+		return in_array('application/json', $parts, true);
```

Comment:
```suggestion
		return in_array('application/json', explode(';', $this->header('Content-Type')->getValue()), true);
```

use one line

Reply:
@mostafakhudair that's harder to read/understand the flow.